dev-python/archspec/archspec-0.2.2.ebuild

```
@@ -0,0 +1,15 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=poetry
+inherit distutils-r1 pypi
+
+DESCRIPTION="labels for various aspects of a system architecture like CPU, etc."
```

```suggestion
DESCRIPTION="Labels for various aspects of a system architecture like CPU, etc."
```

Bit of a nit, but it looks nicer to start the descriptions with capital letters